src/wp-includes/class-wp-plugin-dependencies.php

```diff
@@ -369,10 +369,16 @@ public static function get_dependency_data( $slug ) {
 	 */
 	public static function display_admin_notice_for_unmet_dependencies() {
 		if ( in_array( false, self::get_dependency_filepaths(), true ) ) {
+			$plugins_url = is_multisite() ? network_admin_url( 'plugins' ) : admin_url( 'plugins' );
 			wp_admin_notice(
-				__( 'There are additional plugin dependencies that must be installed.' ),
+				sprintf(
+					/* translators: 1: The start of a link to the plugins page, 2: The end of the link. */
+					__( 'Some required plugins are missing. Please %1$sreview your plugins%2$s for those that cannot be activated.' ),
+					'<a href="' . esc_url( $plugins_url ) . '">',
+					'</a>'
```

Comment:
We don't really do that when there are links. Just use HTML.

```suggestion
					/* translators: %s: Link to the plugins page. */
					__( 'Some required plugins are missing. Please <a href="%s">review your plugins</a> for those that cannot be activated.' ),
					esc_url( $plugins_url ),
```